test/libsolidity/ASTJSON/address_payable_legacy.json

```diff
@@ -40,6 +40,7 @@
           "attributes":
           {
             "constant": false,
+            "documentation": null,
```

Comment:
Can you avoid adding it if it is null?

Reply:
@chriseth: I noticed that too, but I think it's not too bad in having it to see that there is an optional documentation. If it's not there, one may there there'll ever be one. At least that's my argument in keeping it. What do you think?

Reply:
I'm just saying this because we have an issue (that I think has been decided) to not have null-valued fields in general.

Reply:
Was this handled?